ccmlib/common.py

```diff
@@ -730,15 +730,15 @@ def validate_install_dir(install_dir):
             raise ArgumentError(f'{install_dir} does not appear to be a cassandra or dse installation directory')
 
 
-def pid_listening_on(addr: str, port: int):
+def pids_listening_on(addr: str, port: int) -> List[int]:
     '''get pid of the process listening on the given port'''
-    pid = -1
+    pids = []
```

Comment:
nit, could update the docstring accordingly.

Reply:
change the docstring